CHANGELOG.md

```diff
@@ -2,6 +2,100 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.18 - April 22, 2021
+## Breaking changes ⚠️
+* ResourceOptions contains tile store instance (TileStore API). Tile store usage is enabled by default, 'ResourceOptions.tileStoreEnabled' flag is introduced to disable it.
+* Rename MapView plugin extension functions. ([#272](https://github.com/mapbox/mapbox-maps-android/pull/272))
+  - mapView.getAnnotationPlugin() -> mapView.annotations
+  - mapView.getGesturesPlugin() -> mapView.gestures
+  - mapView.getOverlayPlugin() -> mapView.overlay() // using function here because of experimental annotation
+  - mapView.getLocationComponentPlugin() -> mapView.location
+  - mapView.getCameraAnimationsPlugin() -> mapView.camera
+  - mapView.getAttributionPlugin() -> mapView.attribution
+  - mapView.getCompassPlugin() -> mapView.compass
+  - mapView.getLogoPlugin() -> mapView.logo
+  - mapView.getScaleBarPlugin() -> mapView.scalebar
+* Remove deprecated location plugin ([#276](https://github.com/mapbox/mapbox-maps-android/pull/276))
+* Add feature sdk initialisation ([#269](https://github.com/mapbox/mapbox-maps-android/pull/269))
+  - Load the Mapbox Street style by default if user doesn't load any style before the onStart lifecycle event.
```

Comment:
@Chaoba we are now having that entry both in breaking changes and features, is this intended?